scss/_input-group.scss

```diff
@@ -125,7 +125,7 @@
 .input-group-btn:first-child > .dropdown-toggle,
 .input-group-btn:last-child > .btn:not(:last-child):not(.dropdown-toggle),
 .input-group-btn:last-child > .btn-group:not(:last-child) > .btn {
-  @include border-right-radius(0);
+  @include border-radius(0, right);
```

Comment:
I gotcha... instead of (0 $value $value 0), maybe (inherit $value $value inherit). Would that cause any issues? More characters though but that way it would overwrite any other previously written values except for what explicitly being set with the value.